python/chemiscope/input.py

```diff
@@ -156,32 +92,182 @@ def write_input(path, frames, meta=None, extra=None, cutoff=None):
 
         for key in meta.keys():
             if key not in ["name", "description", "authors", "references"]:
-                warnings.warn("ignoring unexpected metadata: {}".format(key))
+                warnings.warn(f"ignoring unexpected metadata: {key}")
```

Comment:
Why ignore instead of incorporate?

Reply:
You mean add the keys to the JSON anyway? Because then we can end up with JSON files containing random keys that do not conform to whatever we expect on the javascript side, making our live harder if/when we want to make changes to the format.